frontend/src/pages/driver-order/driver-order.tsx

```diff
@@ -0,0 +1,113 @@
+            <span className={styles.item}>
+              Distance: {MOCK_TRIP_INFO.distance}km
+            </span>
+            <span className={styles.item}>
+              Time: {MOCK_TRIP_INFO.time} minutes
+            </span>
+            <span className={styles.item}>Price: ${MOCK_TRIP_INFO.price}</span>
+          </div>
+        </div>
+        <div className={styles['order-details']}>
+          <p className={styles.header}>ORDER DETAILS</p>
+          <div>
+            <p className={styles.detail}>
+              <Icon className={styles['user-icon']} iconName="user" /> Customer
+              name:{' '}
+              <span className={styles.value}>
+                {MOCK_ORDER_DETAILS.customerName}
+              </span>
+            </p>
+            <p className={styles.detail}>
+              <Icon className={styles['phone-icon']} iconName="phone" /> Phone:{' '}
+              <span className={styles.value}>{MOCK_ORDER_DETAILS.phone}</span>
+            </p>
+            <p className={styles.detail}>
+              <Icon className={styles['time-icon']} iconName="clock" /> Time:{' '}
+              <span className={styles.value}>{MOCK_ORDER_DETAILS.time}</span>
+            </p>
+            <p className={styles.detail}>
+              <Icon
+                className={styles['location-icon']}
+                iconName="location dot"
+              />{' '}
+              Location:{' '}
+              <span className={styles.value}>
+                {MOCK_ORDER_DETAILS.location}
+              </span>
+            </p>
+            <p className={styles.detail}>
+              <Icon
+                className={styles['destination-icon']}
+                iconName="location dot"
+              />{' '}
+              Destination:{' '}
+              <span className={styles.value}>
+                {MOCK_ORDER_DETAILS.destination}
+              </span>
+            </p>
+            <p className={styles.detail}>
+              <Icon className={styles['car-icon']} iconName="car" /> Cars need
+              to be towed:{' '}
+              <span className={styles.value}>{MOCK_ORDER_DETAILS.cars}</span>
+            </p>
+          </div>
+          <p className={styles['comment-header']}>Comment:</p>
+          <p className={styles.comment}>{MOCK_ORDER_DETAILS.comment}</p>
+          <div className={styles.buttons}>
+            <Button className={styles.button} label={'Decline'} />
+            <Button
+              className={getValidClassNames(styles.button, styles.accept)}
+              label={'Accept'}
+            />
+          </div>
+        </div>
+      </div>
+      <div className={styles.right}>
+        <LoadScript
+          libraries={['places']}
+          googleMapsApiKey={config.ENV.API.GOOGLE_MAPS_API_KEY}
+        >
+          {/* TODO: REMOVE MOCK */}
+          <Map center={{ lat: 21, lng: 22 }} zoom={4} />
+        </LoadScript>
+      </div>
+    </section>
+  );
+};
```

Comment:
As driver shift flow is merged, there should be a button for a driver to end his shift.

Reply:
It is just markup, I think this button can be added with functionality in future